gleam_stdlib/src/list.gleam

```diff
@@ -263,3 +263,12 @@ pub fn range(start, stop) {
   | order:Lt -> [start | range(start + 1, stop)]
   }
 }
+
+external fn duplicate(t, a) -> List(a) = "lists" "duplicate"
+
+pub fn repeat(a, times) {
+  case times <= 0 {
+    | True -> []
+    | False -> duplicate(times, a)
```

Comment:
Can we implement this in Gleam rather than calling out to Erlang? :)

Reply:
sure! wasn't sure what the policy was on using Erlang for some of these

Reply:
reimplemented in Gleam :)